src/metrics/components/ChartSkeletonLoader/ChartSkeletonLoader.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { render, waitForI18n } from '../../../test-utils';
+import { ChartSkeletonLoader } from '../ChartSkeletonLoader/ChartSkeletonLoader';
+
+jest.mock('../../../i18n', () => ({
```

Comment:
This is not right way to get the translation value. We will fix it later. You can check translation key instead. Don't need to mock translation. 

expect(comp.getByText('skeleton_loader_screenreader_text')).toBeInTheDocument();

Reply:
> This is not right way to get the translation value. We will fix it later. You can check translation key instead. Don't need to mock translation.
> 
> expect(comp.getByText('skeleton_loader_screenreader_text')).toBeInTheDocument();

Yes Ajay, here also i just tried the trial and check method for test case by adding mocking translation. i will modify it to our common approach by expecting the test on key .